BPRBlazor/Pages/CreateArchitecturalModelForm.razor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BPRBlazor.Models;
+
+namespace BPRBlazor.Pages;
+
+public partial class CreateArchitecturalModelForm : ComponentBase
+{
+    private ArchitecturalModel _model = new();
+    private string _resultMessage = "";
+    private string _createModelResultCss = "";
+
+    private void AddArchitecturalComponent()
+    {
+        var component = new ArchitecturalComponent()
+        {
+            Id = _model.Components.Any() ? _model.Components.Max(c => c.Id) + 1 : 0
```

Comment:
Wouldn't we want to have a serial or guid instead of this?

Reply:
The id here is only really used as part of the UI model in order to detect which component was clicked on. Could change it to a guid if you prefer, though ultimately I don't think there's much of a point as this id will disappear when converting it from the frontend model to the backend.

Reply:
Ah... thought that would be the one passed to db. All goodie if you prefer this one